### ISSUE TITLE
Sorry, we ran into a problem

### ISSUE THREAD

**Issue by MrM40** [MicrosoftDocs/visualstudio-docs#638]
Cannot login in Visual Studio....."Sorry, we ran into a problem"
Then it is a little difficult to report a problem! Grrrrr
KISS

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d648cd73-34e8-1a47-35cc-3c5a1ff9f8f5
* Version Independent ID: a02d528a-85f7-00f6-cf00-3416713ed4d9
* [Content](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio-2017#sign-in-to-visual-studio)
* [Content Source](https://github.com/Microsoft/visualstudio-docs/blob/master/docs/ide/how-to-report-a-problem-with-visual-studio-2017.md)
* Service: unspecified

**Comment by gewarren** [comment]
@MrM40 Sorry for the frustration. If you check out the [Alternate reporting](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio-2017#alternate-reporting-a-namealternatereportinga) section of that topic, it gives some other options for reporting issues - e.g. the VS installer, or directly to the developer community website. Hope that helps!

**Comment by gewarren** [comment]
@TerryGLee do you think we need to make the Alternate reporting section more prominent?

**Comment by TerryGLee** [comment]
@MrM40 We've updated the text on the "How to Report a Problem" page to read "Reporting alternatives", and we've updated the information about how to use the Visual Studio Developer Community to search for problems and solutions. The text fixes to the "How to Report a Problem" page will go live tomorrow by noon. 

But to further address the problem that you ran in to, there is a similar problem/solution posted on the Developer Community that looks like it might be helpful, here: [Sorry, we ran into a problem](https://developercommunity.visualstudio.com/content/problem/201565/sorry-we-ran-into-a-problem.html).  

And as my colleague remarked earlier, we're sorry for the frustration you experienced. We hope that our response has been helpful. Thanks again for your feedback!
